packages/bridge/src/hooks/useLockedFundsAccounts.tsx

```diff
@@ -0,0 +1,95 @@
+import {useEffect, useState} from "react";
+import {contexts} from "@oyster/common";
+import * as BufferLayout from 'buffer-layout'
+import {WORMHOLE_PROGRAM_ID} from "../utils/ids";
+import BN from "bn.js";
+import {ASSET_CHAIN, getAssetAmountInUSD, getAssetName, getAssetTokenSymbol} from "../utils/assets";
+
+const { useConnection } = contexts.Connection;
+
+interface ParsedData {
+  amount: number,
+  rawAmount: string,
+  parsedAssetAddress: string,
+  parsedAccount: any,
+  assetDecimals: number,
+  tokenName: string,
+  tokenSymbol: string,
+  sourceAddress: string,
+  targetAddress: string,
+  amountInUSD: number,
+}
+
+export const useLockedFundsAccounts = () => {
+  const connection = useConnection();
+
+  const [lockedSolanaAccounts, setLockedSolanaAccounts] = useState<ParsedData[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  useEffect(() => {
+    const queryTxs = async () => {
+      setLoading(true);
+      const programAccounts = await connection.getProgramAccounts(
+        WORMHOLE_PROGRAM_ID
+      );
+      const dataLayout = BufferLayout.struct([
```

Comment:
can you move layouts to separate file in next PR? We will need to expose all the operations as library